apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/get/GetBlockchainStatus.java

```diff
@@ -53,9 +53,15 @@ public JSONObject processRequest(HttpServletRequest req) {
         response.put("version", Constants.VERSION.toString());
         response.put("time", timeService.getEpochTime());
         Block lastBlock = lookupBlockchain().getLastBlock();
-        response.put("lastBlock", lastBlock.getStringId());
-        response.put("cumulativeDifficulty", lastBlock.getCumulativeDifficulty().toString());
-        response.put("numberOfBlocks", lastBlock.getHeight() + 1);
+        if (lastBlock != null) { // TODO: YL I hope that is temporary decision to prevent NPE
```

Comment:
Better throw exception

Reply:
That's for UI only... during starting up node while no genesis/shard snapshot imported